src/osiris/pipelines/transformations.py

```diff
@@ -79,8 +80,8 @@ def process(self, element, *args, **kwargs) -> List[Tuple]:
             file_content = self.datasets.read_file(file_path)
             processed_events_df = pd.read_parquet(BytesIO(file_content), engine='pyarrow')
 
-            processed_events = pd.concat([processed_events_df, events_df], axis=0)
-            processed_events.drop_duplicates(inplace=True, ignore_index=True)
+            processed_events = pd.concat([events_df, processed_events_df], axis=0)
+            processed_events.drop_duplicates(inplace=True, ignore_index=True, subset=self.index_columns)
```

Comment:
```suggestion
            processed_events.drop_duplicates(inplace=True, ignore_index=True, subset=self.index_columns, keep='first')
```
add keep first to be clear about it.

Reply:
Jeg går ud fra, at hvis self.index_columns er None, så tjekker den på alle columns?

Reply:
Ja lige præcis. Så skal rækkerne være helt identiske for at nogle af dem smides væk.